force-app/main/default/classes/OpportunitySummaryByEmailBatch.cls

```diff
@@ -0,0 +1,43 @@
+public class OpportunitySummaryByEmailBatch implements Database.Batchable<sObject>, Database.Stateful  {
+         
+    public Integer emailsSent = 0;
+
+    public Database.QueryLocator start(Database.BatchableContext bc) {
+        return Database.getQueryLocator(
+       		'SELECT Id, Name, ' +
+            '(SELECT email FROM Contacts WHERE Is_Primary_Contact__c = true LIMIT 1), ' +
+            '(SELECT Amount, isWon, isClosed FROM Opportunities) FROM Account'
```

Comment:
Please select the opportunities that were created/updated in the corresponding month (current month).